Learn/pages/sing_in.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <style>
+        body {
+            font-family: Arial, Helvetica, sans-serif;
+        }
+
+        /* .col-sm-3 {
+            position: fixed;
+            top: 50%;
+            left: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            -ms-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+        }   */
+
+        .btn {
+            font-weight: bold;
+        }
+
+        label {
+            display: inline;
+        }
+
+    </style>
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-sm-3 rounded border border-green p-lg-5 mx-auto my-5">
```

Comment:
1. increase width for this column, as an example use width for google login page.
2. **border-green** - no such class in bootstrap, use border-success, see https://getbootstrap.com/docs/4.4/utilities/borders/
3.  Please correct **p-lg-5** class , no such class in bootstrap, See  https://getbootstrap.com/docs/4.4/utilities/spacing/